src/app/side-bar/list/sub-header-list/sub-header-list.component.scss

```diff
@@ -0,0 +1,39 @@
+@import 'src/assets/styles/variables';
+
+.subheader-container {
+  padding: 8px 4px;
+  border: 1px solid $blue-12;
+  width: 93rem;
+  background-color: $floral-white;
+  margin-left: 24px;
+  margin-bottom: 16px;
+  &__content {
+    height: 40px;
+    margin: 0 16px;
+    &-header {
+      padding-bottom: 12px;
+    }
+
+    &-avatar {
+      background-color: $dodger-blue;
+      color: $white;
+    }
+    &-heading {
+      margin-bottom: -8px;
+    }
+    &-sub-text {
+      font-size: 14px;
+    }
+  }
+  &__search:hover {
+    color: $american-violet;
+    cursor: pointer;
```

Comment:
please make utility class for `cursor: pointer;`

Reply:
Resolved..............Thank you